Source/WebKit/NetworkProcess/storage/CacheStorageDiskStore.cpp

```diff
@@ -275,12 +275,11 @@ static std::optional<RecordHeader> decodeRecordHeader(std::span<const uint8_t> h
     };
 }
 
-static std::optional<StoredRecordInformation> readRecordInfoFromFileData(const FileSystem::Salt& salt, const Vector<uint8_t>& buffer)
+static std::optional<StoredRecordInformation> readRecordInfoFromFileData(const FileSystem::Salt& salt, const std::span<const uint8_t> fileData)
```

Comment:
I think we can drop with `const` with the span. We don't usually pass parameters as const when they are passed by value.